### PR TITLE
About team page

### PR DESCRIPTION
**Pull request by sinkoosha**

### CODE REVIEW

File: src/components/AboutTeam/UserCard.js [hunk 1]
```diff
@@ -0,0 +1,46 @@
+import Image from "next/image";
```
Comment (ehsangazar) [inline src/components/AboutTeam/UserCard.js]: Please make it a different component, like UserCard/UserCard

File: src/pages/index.js [hunk 2]
```diff
@@ -30,8 +31,9 @@ export default function Home() {
         <link rel="icon" href="/MocklandLogo.ico" />
       </Head>
       <main>
-        <Hero/>
-        <Container>
+        <Hero />
+        <AboutTeam />
+        {/* <Container>
           <Box display={"grid"} alignItems={"center"}>
```
Comment (ehsangazar) [inline src/pages/index.js]: Remove the commented section please

**Review by ehsangazar — COMMENTED** [review]
Two comments

File: src/pages/index.js (hunk 2 above)
Comment (sinkoosha) [reply src/pages/index.js]: fix it https://github.com/PersiaJS/mockland.dev/commit/9f933461e05b8bcde78cfdee51b90001b4b943ae

**Review by ehsangazar — APPROVED** [review]

**Pull request merged**